app/page.tsx

```diff
@@ -33,7 +33,7 @@ export default function IndexPage() {
             />
           </Link>
         </div>
-        <div className="m-auto flex h-[364px] w-full max-w-[280px] items-center justify-center md:m-0 md:h-full md:w-full lg:max-w-[542px]">
+        <div className="m-auto flex h-[364px] w-full max-w-[280px] items-center justify-center md:m-0 md:h-full md:w-full lg:max-w-[400px]">
           <Image src={PSELogo} alt="pselogo" style={{ objectFit: "cover" }} />
```

Comment:
```suggestion
          <Image src={PSELogo} alt="pselogo" className="object-cover h-{image height in px} w-full"/>
```
Think you need to set the height so the aspect ratio is respected? Not sure why the switch to using `style` here either.

Reply:
because now you have to use style on next/image.
and im already setting the w and h on div above